src/client/App.jsx

```diff
@@ -33,7 +36,7 @@ window.onload = () => {
 
     const appHistory = useScroll(useRouterHistory(createBrowserHistory))();
 
-    ReactDOM.render(
+    ReactDOM.hydrate(
```

Comment:
What's the rationale for this change? Just curious.

Reply:
Following React's preferred protocol: https://reactjs.org/docs/react-dom.html#hydrate
Not a fan?
Thank you so much for giving our PRs an overview! We are so grateful for the input.

Reply:
No, that's fine. It just wasn't available before and didn't know that was the prefered API now.